UnitsNet/UnitDefinitions/Duration.json

```diff
@@ -162,6 +162,18 @@
           "Abbreviations": [ "нс" ]
         }
       ]
+    },
+    {
+      "SingularName": "Cycle",
```

Comment:
I think maybe you are mixing domains here. Duration, to me, is in the time domain, while the factor 0.02 from 1/50 Hz, is from the electrical engineering domain. 

So we could add this as `UtilityFrequency` with units `CyclesPerSecond50` and `CyclesPerSecond60` instead, but I don't see it makes sense to convert between the two units.

I think instead a better approach is to use `Frequency`, which is already in the lib.
So in the US, they have a utilty frequency of 60 Hertz, and you could obtain the time period by something like `TimeSpan period = Frequency.FromHertz(60).Period`.

The `.Period` property does not currently exist, but that could easily be added to `Frequency.extra.cs` partial class.
I chose `TimeSpan` because `Duration` kind of just duplicates `TimeSpan` and is a candidate to be removed from the lib.